commands/admin/adjusttime.js

```diff
@@ -0,0 +1,53 @@
+module.exports.load = (client) => {
```

Comment:
@pianiststickman I was trying to be clever here and reduce the duplication between `addtime` and `deltime`, whose implementations are basically the same.